content/home/marketing/learn-once-write-anywhere.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: Learn Once, Write Anywhere
+title: एक बार जानें, कहीं भी लिखें
 order: 2
 ---
 
-We don't make assumptions about the rest of your technology stack, so you can develop new features in React without rewriting existing code.
+हम आपके अन्य तकनिकी स्टैक के बारे में धारणा नहीं बनाते हैं। तो आप मौजूदा कोड को फिर से लिखे बिना रिएक्ट में नई सुविधाएँ बना सकते हैं।
```

Comment:
Are we translating `React` word? 
As you have translated it in this line to `रिएक्ट`
And above file its not translated

Reply:
@saranshkataria I think you missed one word here.
You have translated react in this line.